shenyu-admin/src/main/java/org/apache/shenyu/admin/model/dto/ScalePolicyDTO.java

```diff
@@ -0,0 +1,323 @@
+        this.id = id;
+    }
+
+    /**
+     * Gets the value of sort.
+     *
+     * @return the value of sort
+     */
+    public @NotNull Integer getSort() {
+        return sort;
+    }
+
+    /**
+     * Sets the sort.
+     *
+     * @param sort sort
+     */
+    public void setSort(@NotNull final Integer sort) {
+        this.sort = sort;
+    }
+
+    /**
+     * Gets the value of status.
+     *
+     * @return the value of status
+     */
+    public @NotNull Integer getStatus() {
+        return status;
+    }
+
+    /**
+     * Sets the status.
+     *
+     * @param status status
+     */
+    public void setStatus(@NotNull final Integer status) {
+        this.status = status;
+    }
+
+    /**
+     * Gets the value of num.
+     *
+     * @return the value of num
+     */
+    public Integer getNum() {
+        return num;
+    }
+
+    /**
+     * Sets the num.
+     *
+     * @param num num
+     */
+    public void setNum(final Integer num) {
+        this.num = num;
+    }
+
+    /**
+     * Gets the value of beginTime.
+     *
+     * @return the value of beginTime
+     */
+    public String getBeginTime() {
+        return beginTime;
+    }
+
+    /**
+     * Sets the beginTime.
+     *
+     * @param beginTime beginTime
+     */
+    public void setBeginTime(final String beginTime) {
+        this.beginTime = beginTime;
+    }
+
+    /**
+     * Gets the value of endTime.
+     *
+     * @return the value of endTime
+     */
+    public String getEndTime() {
+        return endTime;
+    }
+
+    /**
+     * Sets the endTime.
+     *
+     * @param endTime endTime
+     */
+    public void setEndTime(final String endTime) {
+        this.endTime = endTime;
+    }
+
+    /**
+     * builder method.
+     *
+     * @return builder object.
+     */
+    public static ScalePolicyDTO.ScalePolicyDTOBuilder builder() {
+        return new ScalePolicyDTO.ScalePolicyDTOBuilder();
+    }
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        ScalePolicyDTO that = (ScalePolicyDTO) o;
+        return Objects.equals(id, that.id)
+                && Objects.equals(sort, that.sort)
+                && Objects.equals(status, that.status)
+                && Objects.equals(num, that.num)
+                && Objects.equals(beginTime, that.beginTime)
+                && Objects.equals(endTime, that.endTime);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, sort, status, num, beginTime, endTime);
+    }
+
+    public static final class ScalePolicyDTOBuilder {
+
+        private @Existed(provider = ScalePolicyMapper.class, nullOfIgnore = true, message = "scale policy is not existed") String id;
```

Comment:
the annotation location is ugly

Reply:
`
@Existed(provider = ScalePolicyMapper.class, nullOfIgnore = true, message = "scale policy is not existed")
private String id;
`